packages/xo-vmdk-to-vhd/src/vmdk-read.js

```diff
@@ -307,25 +306,30 @@ export class VMDKDirectParser {
 }
 
 export async function readVmdkGrainTable (fileAccessor) {
-  let headerBuffer = await fileAccessor(0, 512)
-  let grainAddrBuffer = headerBuffer.slice(56, 56 + 8)
+  const getLongLong = (buffer, offset) =>
+    new DataView(buffer.slice(offset, offset + 8)).getUint32(0, true)
```

Comment:
Why `offset + 8` and not `offset + 4`?

Why not simply `buffer.readUInt32LE(offset)`?